app.js

```diff
@@ -8,7 +8,6 @@ if (process.env.NEW_RELIC_LICENSE_KEY) {
 
 const compression = require('compression')
 const cookieParser = require('cookie-parser')
-const cookieSession = require('cookie-session')
```

Comment:
express vs cookieSession:
express session seems more storage agnostic? Its used by several examples; not sure that examples couldn't be adapted to use what we already have, though

Reply:
`cookie-session` was definitely in the example usage in the past. If this had changed in more recent years, we can consider updating (in a separate PR)